cc/trees/proxy_impl.cc

```diff
@@ -795,6 +795,9 @@ void ProxyImpl::ScheduledActionCommit() {
 
   auto* commit_state = data_for_commit_->commit_state.get();
   auto* unsafe_state = data_for_commit_->unsafe_state.get();
+  recordreplay::Assert("[Run-1229] ProxyImpl::ScheduledActionCommit %d %d",
+                       commit_state->source_frame_number,
+                       commit_state->trace_id);
```

Comment:
`trace_id` is `uint64_t`, so it needs a different format specifier, e.g. `%llu`.

(AFAIK, printing a 64 bit value with a 32 bit specifier is UB)

Reply:
Argh, I miss Rust yelling at me when I do stupid things

Reply:
Funnily enough, I only just learned that our driver/linker build has warnings enabled for exactly this  😄